vnext/docs/NativeModules.md

```diff
@@ -0,0 +1,54 @@
+# Native Modules Guide for React Native Windows (vnext)
+
+React Native supports interop between JavaScript and native code through the use of native modules.
+
+Custom native modules provide methods and constants implemented in native language that are exposed for use in JavaScript. For example, if you needed access to a thrid-party native image processing library, you could implement a custom native module as a wrapper to expose that library's code to your React Native app.
+
+Custom view managers are specialized native modules that expose native UI controls for use as React Native components. For example, if you wanted to use a third-party XAML control in your React Native app you would create a custom view manager to register that control as a component that you could use then in your React Native app.
+
+## Current Support
+
+| UWP Host Application | Custom Native Modules | Custom View Managers |
+|:---------------------|:---------------------:|:--------------------:|
+| C++/WinRT            | Yes (C++/WinRT only)  | Yes (C++/WinRT only) |
+| C#, C++/CX           | Yes (C#, C++/CX only) | Not yet              |
```

Comment:
Can you add links to issues that tackle support for the ones in the table that have caveats?